<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogThresholding</class>
 <widget class="QDialog" name="DialogThresholding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>maximum</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>threshold</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>binary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>binary_inverted</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>truncated</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>to_zero</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>to_zero_inverted</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>otsu</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSlider" name="horizontalSliderMaximum">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="sliderPosition">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSlider" name="horizontalSliderThreshold">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="sliderPosition">
        <number>128</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
